dev_guide/builds/build_strategies.adoc

```diff
@@ -525,3 +525,42 @@ spec:
     file to use, relative to the source `contextDir`.
     If `contextDir` is omitted, it defaults to the root of the repository.
     If `jenkinsfilePath` is omitted, it defaults to *_Jenkinsfile_*.
+
+[[jenkins-pipeline-strategy-environment]]
+=== Environment Variables
+
+To make environment variables available to the
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline build]
+process, you can add environment variables to the `jenkinsPipelineStrategy` definition
+of the `BuildConfig`.
+
+The environment variables defined there are set as parameters on the job in Jenkins associated with
+the `BuildConfig`. 
+
+For example:
+
+[source,yaml]
+----
+jenkinsPipelineStrategy:
+...
+  env:
+    - name: "FOO"
+      value: "BAR"
+
+----
+
+A couple of details regarding the interaction between Jenkins job parameters and `BuildConfig` environment variables:
+
+1. The value of environment variable in the `BuildConfig` becomes the default value of the parameter in the Jenkins job
+2. The environment variable values in the `BuildConfig` will override any changes to the job parameter default values you make
+from the Jenkins console when starting the job via `oc start-build`; if you start the job from the Jenkins console instead, you 
+can then control the values from the Jenkins console
```

Comment:
this left me confused.  

I think the intent is:  The values provided in the build config take precedence over the values specified in the jenkins job definition.  You can override the values on a per build basis by either using the -e argument with oc start-build, or by using the build with parameters mechanism in the jenkins web console.  (or presumably by specifying env vars when starting a pipeline build via the openshift web console, but that's probably not possible yet)